ui/app/src/Views/AlertList/Views.elm

```diff
@@ -1,44 +1,51 @@
 module Views.AlertList.Views exposing (view)
 
 import Alerts.Types exposing (Alert, AlertGroup, Block)
-import Views.AlertList.Types exposing (AlertListMsg(FilterAlerts))
 import Views.AlertList.Filter exposing (silenced, receiver, matchers)
 import Views.AlertList.FilterBar
 import Html exposing (..)
 import Html.Attributes exposing (..)
 import Utils.Date
-import Utils.Types exposing (Filter)
-import Utils.Views exposing (..)
-import Types exposing (Msg(MsgForAlertList, Noop, CreateSilenceFromAlert, UpdateFilter, AddLabel))
+import Utils.Types exposing (ApiResponse(..), Filter)
+import Utils.Filter
+import Utils.Views exposing (buttonLink, onClickMsgButton, listButton)
+import Views.AlertList.Types exposing (AlertListMsg(..), Model)
+import Types exposing (Msg(Noop, CreateSilenceFromAlert, MsgForAlertList))
+
+
+view : Model -> Filter -> Html Msg
+view { alertGroups, matchers, matcherText, backspacePressed } filter =
+    div []
+        [ Views.AlertList.FilterBar.view matchers matcherText backspacePressed
+        , case alertGroups of
+            Success groups ->
```

Comment:
I very much like moving this peace of logic from the main view function into the AlertList view function 👍